projects/systelab-translate/package.json

```diff
@@ -7,7 +7,7 @@
     "Translate",
     "i18n"
   ],
-  "version": "7.0.6",
+  "version": "7.1.0",
```

Comment:
Isn't it a breaking change? Should we consider a 8.0.0?

Reply:
Totally right..I chose minor but major could be better. I will also add the breaking change to readme file